### DISCUSSION TITLE
Device icon for custom converter?

### DISCUSSION THREAD

**Discussion by DoraDeu** [Koenkk/zigbee2mqtt#15591] [Devices]
I currently use a custom converter in ioBroker for one heating thermostat that is not included to the standard yet (Avatto me167).
Is it possible to apply a custom icon to a custom converter?
I would to show the right device icon instead of the standard "not found" icon in the zigbee adapter.

**Comment by fulup-bzh** [comment]
Placing your custom image within "zigbee2mqtt-frontend/dist" and updating your converter with "icon: image_path" should do the job

Image path is relative to your frontend server root
```
    fingerprint: tuya.fingerprint('TS0601', ['_TZE200_dnz6yvl2']),
    model: 'MOES-smoke',
    icon: '/icons/MOES-smoke.jpg',
    vendor: 'TuYa',
    description: 'Smoke Detector',
```

In my case with a default install within /opt
```
mkdir /opt/zigbee2mqtt/node_modules/zigbee2mqtt-frontend/dist/icons 
cp MOES-smoke.jpg /opt/zigbee2mqtt/node_modules/zigbee2mqtt-frontend/dist/icons 
```

**Reply by DoraDeu** [reply]
Thank you!
I checked that out but it does not work for me as in iobroker the icons are not pulled from the local installation but from from the zigbee2mqtt website.
So I can define whatever I want in the external converter - the mechanism to assign an icon to the device for the zigbee adapter is different there.

**Reply by lux73** [reply]
wouldn't be the ioBroker Website your correct Contact? can't imagine what z2m could do in this Case for you

**Reply by DoraDeu** [reply]
Hm, it is not always easy to identify the right party in this zigbee topics as ioBroker uses zigbee2mqtt and I thought there could be a standard way how to properly assign an icon to a device that is included with a custom converter.
However, I asked this question in the ioBroker forum as well but nobody could or wanted to answer it.

**Reply by lux73** [reply]
ah okay. Thanks for clarify! i wouldn't know that ioBroker even uses z2m ;-)

**Comment by fulup-bzh** [comment]
sorry as you posted your request on zigbee2mqtt I thought you where using the same code.
